x-pack/plugins/security_solution/public/management/links.test.ts

```diff
@@ -32,6 +32,11 @@ describe('links', () => {
   let getPlugins: (roles: string[]) => StartPlugins;
   let fakeHttpServices: jest.Mocked<HttpSetup>;
 
+  const getLinksWithout = (...excludedLinks: SecurityPageName[]) => ({
```

Comment:
Would be worth using the `without` function from `lodash`?

Reply:
I wasn't familiar with `without` so thanks for the suggestion! However, it looks like that `without` uses equality comparison on objects, so I don't think it's possible to filter out objects based on their attributes, `link.id` here.